backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ProjectDatabaseHandler.java

```diff
@@ -221,12 +224,29 @@ public RequestStatus updateProject(Project project, User user) throws SW360Excep
         }
     }
 
-    public void updateProjectForObligations(String id, Map<String, ObligationStatusInfo> obligationStatusMap) {
-        log.info("Updating linked obligations of project: " + id + ", as previously accepted CLI file was changed.");
-        Project project = repository.get(id);
-        project.setLinkedObligations(obligationStatusMap);
+    public ProjectObligation getLinkedObligations(String obligationId, User user) throws TException {
```

Comment:
We can remove the additional parameter `user`, if there is no significant here.

Reply:
There is no significance of `user` object as of now.. But, maybe it will be useful when `REST` end point is available for project obligation.